### ISSUE TITLE
New translation: Norwegian Nynorsk

### ISSUE THREAD

**Issue by tersor** [squidfunk/mkdocs-material#1053]
``` jinja
{% macro t(key) %}{{ {
  "language": "nn",
  "clipboard.copy": "Kopier til utklippstavla",
  "clipboard.copied": "Kopiert til utklippstavla",
  "edit.link.title": "Rediger denne sida",
  "footer.previous": "Førre",
  "footer.next": "Neste",
  "meta.comments": "Kommentarar",
  "meta.source": "Kjelde",
  "search.language": "nn",
  "search.placeholder": "Søk",
  "search.result.placeholder": "Skriv søkeord",
  "search.result.none": "Ingen treff",
  "search.result.one": "1 treff",
  "search.result.other": "# treff",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Gå til innhald",
  "source.link.title": "Gå til kjelde",
  "toc.title": "Innhaldsliste"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Added in 611dc97

**Comment by squidfunk** [comment]
Released as part of 4.2.0.
